CHANGELOG.md

```diff
@@ -1,6 +1,71 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.8.2 (2018-12-19)
+
+This holiday season release fixes a handful of annoyances, and adds an
+experimental `--watch` flag for following the progress of `fluxctl
+release`.
+
+### Fixes
+
+- Respect proxy env entries for git operations
+  [weaveworks/flux#1556][#1556]
+- Only push the "sync tag" when the synced revision has changed,
+  avoiding spurious notifications
+  [weaveworks/flux#1605][#1605]
+- Return any sync errors for workloads in the ListControllers API
+  [weaveworks/flux#1521][#1521]
+
+### Improvements
+
+- The experimental flag `fluxctl release --watch` shows the rollout
+  progress of workloads in the release
+  [weaveworks/flux#1525][#1525]
+- The example manifests now include resource requests, to hepl
```

Comment:
'hepl' -> 'help'

Reply:
Always gotta have a typo, it's basically a law of nature :-S